comments-bundle/src/Resources/contao/classes/Comments.php

```diff
@@ -43,10 +42,9 @@ public function addCommentsToTemplate(FrontendTemplate $objTemplate, \stdClass $
 
 		$objTemplate->comments = array(); // see #4064
 
-		// Tag the response
+		// Tag the comment (see #2137)
 		if (System::getContainer()->has('fos_http_cache.http.symfony_response_tagger'))
 		{
-			/** @var ResponseTagger $responseTagger */
 			$responseTagger = System::getContainer()->get('fos_http_cache.http.symfony_response_tagger');
 			$responseTagger->addTags(array(sprintf('contao.comments.%s.%s', $strSource, $intParent)));
```

Comment:
@Toflar Did you use a different namespace here on purpose? Or should this be `contao.db.`?

Reply:
This is different on purpose because of the source. Also see https://github.com/contao/contao/blob/master/comments-bundle/src/Resources/contao/dca/tl_comments.php#L788

Reply:
I am not sure whether this is correct. If I delete a comment in the back end, Contao will automatically invalidate `['contao.db.tl_comments', 'contao.db.tl_comments.'.$id]`, so would it not make more sense to add a tag for each comment?

Reply:
If you're the first one to comment on `tl_news.12`? How is that invalidated when the comment is published?

Reply:
True. So we have to keep tagging the source record. 👍🏻 

Do you still want me to add tags for each individual comment (`contao.db.tl_comment.ID`) in case someone wants to build a "latest comments" module or something alike? Because there is no hook in the `addCommentsToTemplate()` method which could be used to add the functionality in a third-party bundle.

Reply:
I see, that makes sense I guess?

Reply:
Added in 601247f2eedcfde9d2cc95bd1b764608b2fc6b26.

Last question: Since `tl_comments.source` always relates to a DB table, why do we not use the `contao.db.` prefix instead of the `contao.comments.` prefix? After all, it is a kind of "dynamic ptable" mechanism here as well.

Reply:
I don't know...I just went for this and nobody complained :) I wouldn't change it though as someone might already rely on it.

Reply:
Valid point. I would still like to change it in one of the next feature releases. But I will create a separate ticket for that.